### PR TITLE
docs: add @samgkigotho as a contributor

### PR DESCRIPTION
**Pull request by samgkigotho**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [x] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,9 +43,8 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
     </tr>
     <tr>
-      <td align="center" valign="top" width="14.28%"><a href="https://adiati.com/"><img src="https://avatars.githubusercontent.com/u/45172775?v=4?s=100" width="100px;" alt="Ayu Adiati"/><br /><sub><b>Ayu Adiati</b></sub></a><br /><a href="#question-adiati98" title="Answering Questions">💬</a> <a href="#blog-adiati98" title="Blogposts">📝</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=adiati98" title="Code">💻</a> <a href="#content-adiati98" title="Content">🖋</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=adiati98" title="Documentation">📖</a> <a href="#example-adiati98" title="Examples">💡</a> <a href="#mentoring-adiati98" title="Mentoring">🧑‍🏫</a> <a href="#talk-adiati98" title="Talks">📢</a> <a href="#translation-adiati98" title="Translation">🌍</a></td>
```
Comment (BekahHW) [inline README.md:47]: It looks like you're deleting Ayu's profile. Please update so she remains on the Readme.

File: README.md [hunk 2]
```diff
@@ -44,9 +43,8 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/samgkigotho"><img src="https://avatars.githubusercontent.com/u/116047791?v=4?s=100" width="100px;" alt="Samgkigotho"/><br /><sub><b>Samgkigotho</b></sub></a><br /><a href="#blog-samgkigotho" title="Blogposts">📝</a> <a href="https://github.com/TejasQ/opensauced-guestbook/issues?q=author%3Asamgkigotho" title="Bug reports">🐛</a> <a href="#content-samgkigotho" title="Content">🖋</a> <a href="#data-samgkigotho" title="Data">🔣</a> <a href="#design-samgkigotho" title="Design">🎨</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=samgkigotho" title="Documentation">📖</a> <a href="#example-samgkigotho" title="Examples">💡</a> <a href="#ideas-samgkigotho" title="Ideas, Planning, & Feedback">🤔</a></td>
```
Comment (BekahHW) [inline README.md:47]: It looks like you're overwriting Edgar's profile. Make sure that his remains and yours is separate.

**Review by BekahHW — CHANGES_REQUESTED** [review]

**Review by samgkigotho — COMMENTED** [review]
Have tried to make the changes -please recheck

File: README.md (hunk 2 above)
Comment (BekahHW) [reply README.md:50]: @samgkigotho this still needs to be resolved. But everything else looks good.

Comment (BekahHW) [reply README.md:50]: I'm still not seeing Edgar here.

image

**Comment by samgkigotho** [comment]
@BekahHW please check if okay.

File: README.md (hunk 2 above)
Comment (BekahHW) [reply README.md]: @samgkigotho did you see my comment above?

Comment (samgkigotho) [reply README.md]: Yes, I have tried to make the changes.

**Comment by BekahHW** [comment]
If you look at the bottom of this exchange, you can see if your changes can be merged in without conflicts. Right now, there are conflicts that still need to be resolved.

merge conflict screen shot


 It looks like another PR was merged in, so your changes are overlapping those. Along with that, I see Edgar now--which is great--but we're losing the emojis. If you look at your code, you'll see this:

image


There are two approaches you can take here:

1. You can pull in the new changes and navigate the merge conflicts; or
2. You can start fresh. Create a new branch and fetch the changes from the main branch. Go through the process again and submit the PR. There aren't any other PRs at this point, so I don't anticipate any merge conflicts if you do it this way.

If you want to pair up, I have office hours in the [Discord](https://discord.gg/dXvGd8mz) on Tuesdays at 12p ET.

**Pull request merged**